### ISSUE TITLE
find two same `Striped64.java` files in guava, is this intentional ?

### ISSUE THREAD

**Issue by mjd507** [google/guava#3183]
https://github.com/google/guava/blob/master/guava/src/com/google/common/cache/Striped64.java
https://github.com/google/guava/blob/master/guava/src/com/google/common/hash/Striped64.java

**Comment by kluever** [comment]
Yep, we don't want them to be public APIs, so the only way to use them in both packages is to make 2 copies.

What I didn't realize is that `LongAdder` actually made its way into Java8. I suspect we can delete our copies of `Striped64` and `LongAdder` in our Java8 branch. @lowasser ?

**Comment by lowasser** [comment]
I think this would break GWT/J2CL.
